packages/messaging/src/controllers/window-controller.ts

```diff
@@ -68,13 +73,6 @@ export class WindowController extends ControllerInterface
    * permission isn't granted.
    */
   getToken(): Promise<string | null> {
-    // Check that the required API's are available
-    if (!this.isSupported_()) {
```

Comment:
The check should stay here - moving it to the module instantiation could break peoples applications.